flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/util/FiniteStormFileSpout.java

```diff
@@ -35,6 +35,10 @@
 	private String line;
 	private boolean newLineRead;
 
+	public FiniteStormFileSpout() {
+		super();
```

Comment:
no necessary. is called automatically. make a single line `public FiniteStormFileSpout() {}`